src/main/kotlin/racingcar/Car.kt

```diff
@@ -0,0 +1,7 @@
+package racingcar
+
+class Car(var movement: Int = 0) {
+    fun move() {
+        this.movement = movement + 1
+    }
+}
```

Comment:
매번 모든 것을 상수로 추출하는지에 대해서는 많은 의견이 있는데요.
학습의 의미로 상수 추출을 의식적으로 해보는것도 좋을것 같습니다!
```suggestion
class Car(var movement: Int = INITIAL_POSITION) {
    fun move() {
        this.movement = movement + MOVEMENT_AMOUNT
    }
}
```

Reply:
넵!